src/todofeature/actiontype.js

```diff
@@ -0,0 +1,10 @@
+export const ADD_TODO = "ADD_TODO";
```

Comment:
We can make single object and add this as Key value and return entire obj as export default, like this
![Screenshot from 2022-04-14 15-13-54](https://user-images.githubusercontent.com/70701808/163359297-d82b4ed7-68df-48db-ac9a-582d01a6c6fe.png)